src/migrations/20191119183341-create-hotel.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  up: (queryInterface, Sequelize) => queryInterface.createTable('hotels', {
+  up: async (queryInterface, Sequelize) => queryInterface.createTable('hotels', {
```

Comment:
Why use async here? I've not seen it applied to our other migration files.

Reply:
That's right removed